sheet/ozymandias/Aldin.qml

```diff
@@ -0,0 +1,501 @@
+
+        permanent.modifiers: [
+            Bonus {
+                source: pointBuy
+                amount: 4
+            },
+            Bonus {
+                source: race
+                amount: 2
+            }
+        ]
+    }
+
+    AbilityScore {
+        id: wisdom
+        name: "Wisdom"
+
+        permanent.modifiers: [
+            Bonus {
+                source: pointBuy
+                amount: 2
+            }
+        ]
+    }
+
+    AbilityScore {
+        id: charisma
+        name: "Charisma"
+
+    }
+
+
+
+
+    Item {
+        id: hp
+        property string prefix: "attr://combat/hitPoints"
+
+        Attribute {
+            id: maxHp
+            name: "Maximum Hit Points"
+            uri: hp.prefix + "/maximum"
+
+            modifiers: [
+                Bonus {
+                    name: constitution.temporary.modifier.name
+                    amount: constitution.temporary.modifier.value*level.value
+                },
+                Bonus {
+                    name: "Level 1 Health Roll (Max)"
+                    amount: 8
+                },
+                Bonus {
+                    name: "Favoured Class"
+                    amount: level.value
+                },
+                Bonus {
+                    source: toughness
+                    amount: Math.max(3, level.value)
+                },
+                Bonus{
+                    name: "Level 2 Health Roll"
+                    amount: 5
+                },
+                Bonus{
+                    name: "Level 3 Health Roll"
+                    amount: 7
+                }
+
+            ]
+        }
+
+        Attribute {
+            id: currentHp
+            name: "Current Hit Points"
+            uri: hp.prefix + "/current"
+
+            modifiers: [
+                Bonus {
+                    name: "Initial Value (Max)"
+                    amount: maxHp.value
+                }
+            ]
+            readOnly: false
+        }
+    }
+
+    Attribute {
+        id: armourClass
+        name: "Armour Class"
+        uri: "attr://combat/armourClass"
+
+        modifiers: [
+            Bonus {
+                name: "Base"
+                amount: 10
+            },
+            Bonus {
+                id: acDexBonus
+                name: "Dexterity"
+                amount: dexterity.temporary.modifier.value
+            },
+            Bonus {
+                id: acArmourBonus
+                name: "Armour (Mage Armor)"
+                amount: 4
+            },
+            Bonus {
+                id: acShieldBonus
+                name: "Buckler"
+                amount: 1
+            },
+            Bonus {
+                id: acDeflectionBonus
+                name: "Hawk Badge"
+                amount: 1
+            },
+
+            Bonus {
+                name: size.name
+                amount: -sizeBonus(size.value)/2
+            }
+
+        ]
+
+        Attribute {
```

Comment:
These two can now be redone properly using BonusTypes. Look at #25 as an example.